### PR TITLE
Freshen up "updateeachrelease" articles

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #27139

### CODE REVIEW

File: docs/csharp/language-reference/includes/langversion-table.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ ms.custom: "updateeachrelease"
 | `preview`                 | The compiler accepts all valid language syntax from the latest preview version.                         |
 | `latest`                  | The compiler accepts syntax from the latest released version of the compiler (including minor version). |
 | `latestMajor` (`default`) | The compiler accepts syntax from the latest released major version of the compiler.                     |
-| `10.0`                    | The compiler accepts only syntax that is included in C# 10 or lower.                                  |
-| `9.0`                     | The compiler accepts only syntax that is included in C# 9.0 or lower.                                   |
+| `10.0`                    | The compiler accepts only syntax that is included in C# 10 or lower.                                    |
+| `9.0`                     | The compiler accepts only syntax that is included in C# 9 or lower.                                     |
```
Comment (gewarren) [inline docs/csharp/language-reference/includes/langversion-table.md:11]: I may be recalling incorrectly that C# 9.0 should now just be C# 9.

File: docs/standard/whats-new/whats-new-in-dotnet-standard.md [hunk 2]
```diff
@@ -1,31 +1,45 @@
...
 - .NET Framework 4.6.1 or later
 - Mono 5.4 or later
 - Xamarin.iOS 10.14 or later
 - Xamarin.Mac 3.8 or later
 - Xamarin.Android 8.0 or later
 - Universal Windows Platform 10.0.16299 or later
 
+## What's new in .NET Standard 2.1
+
+.NET Standard 2.1 adds many new APIs. Some of them are new APIs, and others are existing APIs that help to converge the .NET implementations even further. For a list of the APIs that have been added to .NET Standard 2.1, see [.NET Standard 2.1 vs 2.0](https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.1_diff.md).
```
Comment (tdykstra) [inline docs/standard/whats-new/whats-new-in-dotnet-standard.md:36]: Confusing contradiction

>adds many new APIs ... some of them are new ... others are existing

"new APIs" must mean something different in the first occurrence than it does in the second occurrence, and it isn't clear what "add an existing API" refers to

**Review by tdykstra — APPROVED** [review]
LGTM

File: docs/standard/whats-new/whats-new-in-dotnet-standard.md (hunk 2 above)
Comment (gewarren) [reply docs/standard/whats-new/whats-new-in-dotnet-standard.md:36]:
```suggestion
.NET Standard 2.1 adds many APIs to the standard. Some of them are new APIs, and others are existing APIs that help to converge the .NET implementations even further. For a list of the APIs that have been added to .NET Standard 2.1, see [.NET Standard 2.1 vs 2.0](https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.1_diff.md).
```

**Pull request merged**